src/views/confirm.njk

```diff
@@ -0,0 +1,576 @@
+              }
+            ]
+          }
+        } if model.roe === 'yes',
+        {
+          key: {
+            text: "Experience Controlling Sika Deer"
+          },
+          value: {
+            text: model.sika
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/sika-experience#sika",
+                text: "Change",
+                visuallyHiddenText: "Experience Controlling Sika Deer"
+              }
+            ]
+          }
+        },
+        {
+          key: {
+            text: "Years Experience"
+          },
+          value: {
+            text: model.sikaExperienceYears
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/sika-experience#sika-experience-years",
+                text: "Change",
+                visuallyHiddenText: "Experience Controlling Sika Deer"
+              }
+            ]
+          }
+        } if model.sika === 'yes',
+        {
+          key: {
+            text: "Number Controlled in the last 5 years"
+          },
+          value: {
+            text: model.sikaControlNumber
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/sika-experience#sika-control-number",
+                text: "Change",
+                visuallyHiddenText: "Number Controlled in the last 5 years"
+              }
+            ]
+          }
+        } if model.sika === 'yes',
+        {
+          key: {
+            text: "Experience Controlling Fallow Deer"
+          },
+          value: {
+            text: model.fallow
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/fallow-experience#fallow",
+                text: "Change",
+                visuallyHiddenText: "Experience Controlling Fallow Deer"
+              }
+            ]
+          }
+        },
+        {
+          key: {
+            text: "Years Experience"
+          },
+          value: {
+            text: model.fallowExperienceYears
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/fallow-experience#fallow-experience-years",
+                text: "Change",
+                visuallyHiddenText: "Experience Controlling Fallow Deer"
+              }
+            ]
+          }
+        } if model.fallow === 'yes',
+        {
+          key: {
+            text: "Number Controlled in the last 5 years"
+          },
+          value: {
+            text: model.fallowControlNumber
+          },
+          actions: {
+            items: [
+              {
+                href: pathPrefix + "/fallow-experience#fallow-control-number",
+                text: "Change",
+                visuallyHiddenText: "Number Controlled in the last 5 years"
+              }
+            ]
+          }
+        } if model.fallow === 'yes'
+      ]
+    }) }}
```

Comment:
There's a few wee things in here that could be changed... and one big one.

- The `yes` and `no` answers should really be `Yes` and `No` - they look out of place with leading lower cases.
- **Years experience** doesn't tell you what you've got experience of - in general or specific.
    - **Years experience controlling Red Deer** is more specific.
- **Number Controlled in the last 5 years** is similarly unspecific and also misleading. The question as asked was **What is the average number of ... you have controlled per year over the last 5 years?** That's a 5x difference in what that number's supposed to be.


Now for the biggie...
Overall, this is a bit cluttered...

![image](https://user-images.githubusercontent.com/37802088/97876901-5ce51300-1d14-11eb-8ee7-0ee6f67b3383.png)

In SFO, we implemented something similar by adding a sub-table...

![image](https://user-images.githubusercontent.com/37802088/97876802-3e7f1780-1d14-11eb-8fcd-afb162db07cd.png)

...which could look something like this, if you implemented it...

![image](https://user-images.githubusercontent.com/37802088/97879066-67ed7280-1d17-11eb-8d5e-7ae0ac31ce48.png)

Reply:
yeah I think a sub table would be a good approach for this, like you said it looks a tad cluttered at the moment.

Reply:
Resolved in 676aacf90b98b3e88565700fefe2a6f8986e1bda.